assets/js/src/blocks/team-list/edit.js

```diff
@@ -12,6 +11,9 @@ import {
 	ToggleControl,
 	RangeControl,
 } from '@wordpress/components';
+const {
+	InspectorControls,
+} = wp.blockEditor || wp.editor;
```

Comment:
How bad is the notice really? I think we should stick with `@wordpress/editor` here until `InspectorControls` gets removed and/or we bump the required WP version.

With this neither `wp-editor` nor `wp-block-editor` is added to the list of dependencies.

Reply:
It is a notice in the developer console. In PHP we try to use the new functions even if the deprecated functions still work.